targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1050/TARGET_EVK/PinNames.h

```diff
@@ -182,33 +182,33 @@ typedef enum {
     CONSOLE_TX = GPIO_AD_B0_12,
     CONSOLE_RX = GPIO_AD_B0_13,
 
-    // Arduino Headers
-    D0 = GPIO_AD_B1_07,
-    D1 = GPIO_AD_B1_06,
-    D2 = GPIO_AD_B0_11,
-    D3 = GPIO_AD_B1_08,
-    D4 = GPIO_AD_B0_09,
-    D5 = GPIO_AD_B0_10,
-    D6 = GPIO_AD_B1_02,
-    D7 = GPIO_AD_B1_03,
-    D8 = GPIO_AD_B0_03,
-    D9 = GPIO_AD_B0_02,
-    D10 = GPIO_SD_B0_01,
-    D11 = GPIO_SD_B0_02,
-    D12 = GPIO_SD_B0_03,
-    D13 = GPIO_SD_B0_00,
-    D14 = GPIO_AD_B0_01,
-    D15 = GPIO_AD_B0_00,
-
-    A0 = GPIO_AD_B1_10,
-    A1 = GPIO_AD_B1_11,
-    A2 = GPIO_AD_B1_04,
-    A3 = GPIO_AD_B1_05,
-    A4 = GPIO_AD_B1_01,
-    A5 = GPIO_AD_B1_00,
-
-    I2C_SCL = A5,
-    I2C_SDA = A4,
+    // Arduino Uno Headers
+    ARDUINO_UNO_D0 = GPIO_AD_B1_07,
+    ARDUINO_UNO_D1 = GPIO_AD_B1_06,
+    ARDUINO_UNO_D2 = GPIO_AD_B0_11,
+    ARDUINO_UNO_D3 = GPIO_AD_B1_08,
+    ARDUINO_UNO_D4 = GPIO_AD_B0_09,
+    ARDUINO_UNO_D5 = GPIO_AD_B0_10,
+    ARDUINO_UNO_D6 = GPIO_AD_B1_02,
+    ARDUINO_UNO_D7 = GPIO_AD_B1_03,
+    ARDUINO_UNO_D8 = GPIO_AD_B0_03,
+    ARDUINO_UNO_D9 = GPIO_AD_B0_02,
+    ARDUINO_UNO_D10 = GPIO_SD_B0_01,
+    ARDUINO_UNO_D11 = GPIO_SD_B0_02,
+    ARDUINO_UNO_D12 = GPIO_SD_B0_03,
+    ARDUINO_UNO_D13 = GPIO_SD_B0_00,
+    ARDUINO_UNO_D14 = GPIO_AD_B0_01,
+    ARDUINO_UNO_D15 = GPIO_AD_B0_00,
+
+    ARDUINO_UNO_A0 = GPIO_AD_B1_10,
+    ARDUINO_UNO_A1 = GPIO_AD_B1_11,
+    ARDUINO_UNO_A2 = GPIO_AD_B1_04,
+    ARDUINO_UNO_A3 = GPIO_AD_B1_05,
+    ARDUINO_UNO_A4 = GPIO_AD_B1_01,
+    ARDUINO_UNO_A5 = GPIO_AD_B1_00,
+
+    I2C_SCL = ARDUINO_UNO_A5,
```

Comment:
These lines should be removed as already defined here
https://github.com/ARMmbed/mbed-os/pull/14617

Reply:
@MarceloSalazar I will remove those lines. After the update and commit to my fork, will that automatically be seen in this PR? Or must a follow some procedure to update this PR with new commits?

Thank you!

Reply:
@MarceloSalazar I will update the PR today. Thanks again.